test/go/trivial_integration_test.go

```diff
@@ -167,6 +140,91 @@ func TestCopyDataWithNullInColumn(t *testing.T) {
 	testcase.Run()
 }
 
+func TestCopyDataInFixedSizeBinaryColumn(t *testing.T) {
```

Comment:
The golang integration tests are deprecated. Can you write the tests in the ruby integration test instead? Specifically it can go under https://github.com/Shopify/ghostferry/blob/master/test/integration/types_test.rb.